examples/configuration/echo.lua

```diff
@@ -0,0 +1,7 @@
+local policy_chain = require('apicast.policy_chain').build({ })
+
+policy_chain:insert(policy_chain.load('apicast.policy.echo'))
+
+return {
+	policy_chain = policy_chain
```

Comment:
We will have to start documenting the environment config format. It is Lua, so no JSON schema.